src/scripts/components/dropstrip/dropstrip-store.jsx

```diff
@@ -66,7 +66,7 @@ const DropstripStore = assign({}, EventEmitter.prototype, {
 
   success(filename) {
     dropzoneQueue[filename].completed = true;
-    resoundAPI.get(filename)
+    resoundAPI.get()
```

Comment:
This is almost what I wanted in https://github.com/ProjectResound/planning/issues/49

Instead of a general .get(), when there's a success, we should just prepend the associated AudioItem to the list of items in ExplorerStore. That's because in the future, ExplorerStore may return a sorted or filtered list based on duration, name, etc. so if we just do a .get() it would undo the filtering.